alibi_detect/utils/missing_optional_dependency.py

```diff
@@ -25,7 +25,8 @@
     "tensorflow_probability": 'tensorflow',
     "tensorflow": 'tensorflow',
     "torch": 'torch',
-    "pytorch": 'torch'
+    "pytorch": 'torch',
+    "keops": 'keops',
```

Comment:
One for @mauicv here. I've had to swap `pykeops` for `keops` here (in https://github.com/SeldonIO/alibi-detect/pull/548/commits/9c2da77734c1d30cbb8da8f9312da23a5441c249), as I was unable to import `MMDDrift` when `pykeops` was not installed, due to `BackendValidator` raising the error `KeyError: 'keops'`. 

However, this means the error message is slightly incorrect:
```
ImportError: `keops` not installed. Cannot initialize and run MMDDrift with keops backend. The necessary missing dependencies can be installed using `pip install alibi-detect[keops]`.
```
Since the library is actually called `pykeops` despite our bucket being called `keops`. Is this a limitation of `BackendValidator`?

On a side note, this is an example of why I still wonder if we should have optional deps tests to check the wrapper classes such as `MMDDrift` (`test_mmd.py` was failing when `pykeops` was not installed, however this was not picked up since we don't test this with the optional deps `tox` based tests).

Reply:
As spoken about offline the fix is to add `"pykeops": 'keops'` to the `ERROR_TYPES` dictionary.

Reply:
Just pushed a fix. Thanks Alex!